rs/ethereum/cketh/docs/cketh.adoc

```diff
@@ -40,24 +40,24 @@ Please feel free to reach out by leaving a comment in this link:https://forum.df
 == Deposit: ETH to ckETH
 
 ----
- ┌────┐               ┌───────────────┐              ┌──────┐
- │User│               │Helper Contract│              │Minter│
- └─┬──┘               └───────┬───────┘              └──┬───┘
-   │                          │                         │
-   │deposit(amount, principal)│                         │
-   │─────────────────────────>│                         │
-   │                          │                         │
-   │                          │       get_events        │
-   │                          │<────────────────────────│
-   │                          │                         │
-   │                          │Events(amount, principal)│
-   │                          │────────────────────────>│
-   │                          │                         │
-   │              mint(amount, principal)               │
-   │<───────────────────────────────────────────────────│
- ┌─┴──┐               ┌───────┴───────┐              ┌──┴───┐
- │User│               │Helper Contract│              │Minter│
- └────┘               └───────────────┘              └──────┘
+ ┌────┐                               ┌───────────────┐                                                ┌──────┐
+ │User│                               │Helper Contract│                                                │Minter│
+ └─┬──┘                               └───────┬───────┘                                                └──┬───┘
+   │                                          │                                                           │
+   │depositEth(amount, principal, subaccount) │                                                           │
+   │─────────────────────────────────────────>│                                                           │
+   │                                          │                                                           │
+   │                                          │       get_events                                          │
+   │                                          │<──────────────────────────────────────────────────────────│
+   │                                          │                                                           │
+   │                                          │ReceivedEthOrErc20(0x, user, amount, principal, subaccount)│
```

Comment:
Potentially surround 0x with quotes to emphasize it is a fixed value as opposed to the other parameters.

Reply:
make sense, done.